internal/pkg/levee/levee.go

```diff
@@ -69,15 +77,32 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return nil, nil
 }
 
-func reportSourcesReachingSink(conf *config.Config, pass *analysis.Pass, propagations map[*source.Source]propagation.Propagation, sink ssa.Instruction) {
+func reportSourcesReachingSink(conf *config.Config, pass *analysis.Pass, suppressedNodes suppression.ResultType, propagations map[*source.Source]propagation.Propagation, sink ssa.Instruction) {
 	for src, prop := range propagations {
-		if prop.IsTainted(sink) {
+		if prop.IsTainted(sink) && !isSuppressed(sink.Pos(), suppressedNodes, pass) {
 			report(conf, pass, src, sink.(ssa.Node))
 			break
 		}
 	}
 }
 
+func isSuppressed(pos token.Pos, suppressedNodes suppression.ResultType, pass *analysis.Pass) bool {
+	for _, f := range pass.Files {
+		if pos < f.Pos() || f.End() < pos {
+			continue
+		}
```

Comment:
Prefer `pass.Fset.Position(pos).Filename` to get the filename to avoid the loop.  Check for `pos != 0`.

Reply:
I don't think we can avoid the loop here. If we get the `Filename` then we can use that instead of  `f.Pos()` and `f.End()` to find the appropriate `*ast.File`, but AFAIK there isn't an existing function to get the `*ast.File` from the filename. :thinking:

Reply:
Whoops.  Don't know why I thought those were `pass.Files` were `string`s.  Yeah, disregard.